src/features/hotspots/settings/HotspotDiagnosticReport.tsx

```diff
@@ -122,6 +131,16 @@ const HotspotDiagnosticReport = ({ onFinished }: Props) => {
           ? formatMac(diagnostics.eth)
           : t('generic.unavailable'),
       },
+      {
+        attribute: t('hotspot_settings.diagnostics.disk'),
+        value: diagnostics?.disk
+          ? formatMac(diagnostics.disk)
```

Comment:
Are the values of this `"read only"` and `"disk ok"`? Will formatMac work for those?

Reply:
I fixed this up based on the github issue. Good catch!